src/Microsoft.ML.Dnn/DnnCatalog.cs

```diff
@@ -80,59 +81,53 @@ public static class DnnCatalog
 
         /// <summary>
         /// Performs image classification using transfer learning.
-        /// usage of this API requires additional NuGet dependencies on TensorFlow redist, see linked document for more information.
+        /// Usage of this API requires additional NuGet dependencies on TensorFlow redist, see linked document for more information.
         /// <format type="text/markdown">
         /// <![CDATA[
         /// [!include[io](~/../docs/samples/docs/api-reference/tensorflow-usage.md)]
         /// ]]>
         /// </format>
         /// </summary>
-        /// <param name="catalog"></param>
-        /// <param name="featuresColumnName">The name of the input features column.</param>
-        /// <param name="labelColumnName">The name of the labels column.</param>
-        /// <param name="scoreColumnName">The name of the output score column.</param>
-        /// <param name="predictedLabelColumnName">The name of the output predicted label columns.</param>
-        /// <param name="validationSet">Validation set.</param>
+        /// <param name="catalog">Catalog</param>
+        /// <param name="options">An <see cref="ImageClassificationTrainer.Options"/> object specifying advanced options for <see cref="ImageClassificationTrainer"/>.</param>
 
-        public static ImageClassificationEstimator ImageClassification(
-            this ModelOperationsCatalog catalog,
-            string featuresColumnName,
-            string labelColumnName,
-            string scoreColumnName = "Score",
-            string predictedLabelColumnName = "PredictedLabel",
-            IDataView validationSet = null
-            )
+        public static ImageClassificationTrainer ImageClassification(
+            this MulticlassClassificationCatalog.MulticlassClassificationTrainers catalog,
+            ImageClassificationTrainer.Options options)
         {
-            var options = new ImageClassificationEstimator.Options()
-            {
-                FeaturesColumnName = featuresColumnName,
-                LabelColumnName = labelColumnName,
-                ScoreColumnName = scoreColumnName,
-                PredictedLabelColumnName = predictedLabelColumnName,
-                ValidationSet = validationSet
-            };
+            options.EarlyStoppingCriteria = options.DisableEarlyStopping ? null : options.EarlyStoppingCriteria ?? new ImageClassificationTrainer.EarlyStopping();
```

Comment:
I made a comment on the previous PR that you said you would address in this PR.

[https://github.com/dotnet/machinelearning/pull/4362#discussion_r338160348](https://github.com/dotnet/machinelearning/pull/4362#discussion_r338160348)